packages/overlay/test/overlay-element.test.ts

```diff
@@ -74,6 +79,140 @@ describe('sp-overlay', () => {
         });
     }
 
+    describe.only('`slottable-request` event', () => {
```

Comment:
I assume `.only()` wasn't meant to be checked in?

Reply:
Thanks for catching this. I had thought we'd put linting for this in previously, but it looks like I'll need to add that in a separate PR.

Fixed!